x-pack/plugins/security_solution/common/api/timeline/install_prepackaged_timelines/install_prepackaged_timelines.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+
+import { unionWithNullType } from '../../../utility_types';
+import { ImportTimelinesSchemaRt, TimelineSavedToReturnObjectRuntimeType } from '..';
+
+export const checkTimelineStatusRt = rt.type({
```

Comment:
I wonder if it's better to rename it to `checkPrepackagedTimelines` or `checkImportTimelines` or something a little bit more specific.